lib/routes/searches.js

```diff
@@ -0,0 +1,34 @@
+const router = require('express').Router();
+const Search = require('../model/Search');
+
+router.post('/', (request, response) => {
+  const tournament = request.body.tournament;
+  const user = request.body.user;
+
+  const newSearch = new Search({
+    tournament,
+    user,
+  });
+
+  newSearch
+    .save()
+    .then((newSearch) => response.json(newSearch))
+    .catch((err) => response.json('Error:' + err));
```

Comment:
Happens multiple times:
On error, you return the default HTTP status `200`. The client would think that everything is ok.

You should return an error status code on error.

Maybe

```suggestion
    .catch((err) => response.status(500).json(error));
```

Reply:
Thanks! I fixed it within this file and opened an issue to check the whole project: #47